multi_gpu/one/model.py

```diff
@@ -0,0 +1,147 @@
+####################
```

Comment:
How much of the Python code changes in the 1, 2, and 4 GPU examples? Can we have the number of GPUs be an argument to the Python file?

Reply:
I second this notion a lot of the code is redundant